.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -1330,8 +1330,10 @@ private DateTimeResolutionResult ParseMonthWithYear(string text, DateObject refe
             {
                 var monthStr = match.Groups["month"].Value;
                 var orderStr = match.Groups["order"].Value;
+                var cardinalStr = match.Groups["cardinal"].Value;
 
-                var month = this.config.MonthOfYear[monthStr];
+                var month = match.Groups["month"].Success ? this.config.MonthOfYear[monthStr] :
```

Comment:
Can you add a code comment here to make it easier to understand?

Reply:
added